hail/python/hail/methods/statgen.py

```diff
@@ -609,6 +617,12 @@ def logistic_regression_rows(test, y, x, covariates) -> hail.Table:
 
     .. _EPACTS: http://genome.sph.umich.edu/wiki/EPACTS#Single_Variant_Tests
 
+    Note
+    ----
+    You can include additional row fields from the input :class:`.MatrixTable`
```

Comment:
I prefer to avoid "you".  Change these to:
```
Use the `pass_through` parameter to include additional row fields from matrix table underlying ``x``.  For example, to include an "rsid" field, set ``pass_through=['rsid']`` or ``pass_through=[mt.rsid]``.
```